remote-config.jsonc

```diff
@@ -177,7 +177,7 @@
           "message": "DDEV Swag is available, including t-shirts, mugs, etc. see https://ddev.threadless.com/ - thanks @bmartinez287!"
         },
         {
-          "message": "DDEV allows you to rapidly profile the performance of your PHP code using XHProf with `ddev xhprof on`. Try out https://github.com/ddev/ddev-xhgui !"
+          "message": "DDEV v1.24.0 has in-built support for XHGui, thanks to TYPO3 Community sponsorship."
```

Comment:
I would add "and @tyler36 's long-term stewardship of the feature through its days as an add-on. But I'll bet you wouldn't add that :)

Reply:
```suggestion
          "message": "DDEV v1.24.4+ has built-in support for XHGui for exploring performance issues, thanks to TYPO3 Community sponsorship."
```